### ISSUE TITLE
Allow users to specify different column names for effects

### ISSUE THREAD

**Issue by be-green** [wwiecek/baggr#44]
One of the things that confused me when I first read through the documentation for this package was that the treatment effect was name tau and the variance was sigma_tau. On the rstan getting started page, for example, where they list the 8 schools example, what we call tau_k they call y_i. What we call sigma_tau they call tau! Anyway I just thought it might be good to allow people to list the column name that contains all the different effects instead of making them change the name of their data.frame or data.table, because if they have their own notations/conventions their code might be hard to read. Just a thought!

**Comment by wwiecek** [comment]
I changed the notation within .stan models to 

* theta_k for group-specific effects, 
* mu for hypermean; if we are modelling multiple random vectors (like in "mu & tau" model), I just vectorise mu (so it's "mu[1] and mu[2]" model, with mu[1] being baseline, mu[2] being treatment; that model can work with more than 2 parameters)
* tau for hypersd (same as above, tau can be a vector)

@rmeager Are you happy with using that notation? If yes, at some point we should change the vignettes and inputs to reflect that. I wouldn't rush it, as changing syntax should be done carefully.

**Comment by wwiecek** [comment]
This got pushed past v0.4 as we are still not agreed on what notation to best use. I think it would be good to change it before the next release because the models are beginning to be "set in stone" a bit.

**Comment by wwiecek** [comment]
This will be fixed in #235 (finally!!!)

**Comment by be-green** [comment]
Wow that's a throwback of a github email. Hope all is well!

**Comment by wwiecek** [comment]
Yeah, thanks! I thought you'd find it funny that I am fixing six year old issues, but hey, hobby project!
